templates/index.ftl

```diff
@@ -10,90 +10,63 @@
-                        <img alt="Integration" src="${content.rootpath}landingPage/integrationOverview.png">
-                    </div>
-                    <div class="carousel-item">
-                        <img alt="Compatibility" src="${content.rootpath}landingPage/compatibility.png">
-                    </div>
-                    <div class="carousel-item">
-                        <img alt="Scalability" src="${content.rootpath}landingPage/scalabilityOfOptimizationAlgorithms.png">
-                    </div>
-                </div>
-                <button class="carousel-control-prev" type="button" data-bs-target="#frontPageCarousel" data-bs-slide="prev">
-                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
-                    <span class="visually-hidden">Previous</span>
-                </button>
-                <button class="carousel-control-next" type="button" data-bs-target="#frontPageCarousel" data-bs-slide="next">
-                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
-                    <span class="visually-hidden">Next</span>
-                </button>
+        <h2 class="mb-2">What can OptaPlanner do?</h2>
+        <p>OptaPlanner optimizes plans and schedules with hard and soft constraints
+            to <b>heavily reduce costs</b>, improve <b>service quality</b>, fulfill <b>employee wishes</b>
+            and <b>lower carbon emissions</b>.</p>
+        <@macros.useCaseCards/>
+
+        <div class="row">
+            <div class="col-md-6">
+                <h2>Modern mathematical optimization</h2>
+                <p>OptaPlanner is <strong>a lightweight, embeddable planning engine</strong>.
+                    It enables everyday programmers to solve optimization problems efficiently.
+                    Constraints apply on plain domain objects and can call existing code.
+                    There’s no need to input constraints as mathematical equations.</p>
+                <p>Under the hood, OptaPlanner combines sophisticated Artificial Intelligence optimization algorithms
+                    (such as Tabu Search, Simulated Annealing, Late Acceptance and other metaheuristics)
+                    with very efficient score calculation and other state-of-the-art constraint solving techniques.</p>
+            </div>
+            <div class="col-md-6">
+                <h2>Compatibility</h2>
+                <span>Works directly from:</span>
+                <ul>
+                    <li><a href="${content.rootpath}compatibility/java.html">Java<i class="fab fa-java ms-1"></i></a></li>
+                    <li><a href="${content.rootpath}compatibility/kotlin.html">Kotlin</a></li>
+                    <li><a href="${content.rootpath}compatibility/scala.html">Scala</a></li>
+                    <li><a href="${content.rootpath}compatibility/python.html">Python<i class="fab fa-python ms-1"></i></a></li>
```

Comment:
Please don't. The reason why we made it a separate project was so that we do not have to commit to it.
It's front-page material now?

Reply:
As discussed, we're adding the (Experimental) tag for the frontpage, but keeping it here. cc @Christopher-Chianelli